### ISSUE TITLE
After fresh install Cannot commit transaction when creating a password

### ISSUE THREAD

**Issue by Kassouma** [passbolt/passbolt_api#392]
## After fresh install Cannot commit transaction when creating a password
* Passbolt Version: 3.1.0-1
* Platform and Target:
-- Operating system: Debian 10.9
-- PHP: PHP 7.3.27-1~deb10u1 (cli) (built: Feb 13 2021 16:31:40) ( NTS )
-- Web server: nginx/1.14.2
-- Database server: mysql  Ver 15.1 Distrib 10.3.27-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
-- etc.: followed debian 10 installation guide

### What you did
Followed debian 10 installation guide, used auto install of nginx and ssl cert. Created admin user and logged in.

### What happened
Creating users work. Logging works. If I try to create a password it fails with "Internal error". I enable debug = true in app.php but only found the same log as in  /var/log/passbolt/error.log 

```
==> /var/log/passbolt/error.log <==
2021-04-04 08:17:59 Error: [Cake\Database\Exception\NestedTransactionRollbackException] Cannot commit transaction - rollback() has been already called in the nested transaction (/usr/share/php/passbolt/vendor/cakephp/cakephp/src/Database/Connection.php:578)
Request URL: /resources.json?api-version=v2&contain%5Bpermission%5D=1&contain%5Bfavorite%5D=1

```
### What you expected to happen
That my password is added to my "folder".
Importing kdbx leads to same issue.

### Healthcheck
```
root@pass:/usr/share/php/passbolt# su -s /bin/bash -c "./bin/cake passbolt healthcheck" www-data

     ____                  __          ____  
    / __ \____  _____ ____/ /_  ____  / / /_ 
   / /_/ / __ `/ ___/ ___/ __ \/ __ \/ / __/ 
  / ____/ /_/ (__  |__  ) /_/ / /_/ / / /    
 /_/    \__,_/____/____/_.___/\____/_/\__/   

 Open source password manager for teams
---------------------------------------------------------------
 Healthcheck shell       
---------------------------------------------------------------

 Environment

 [PASS] PHP version 7.3.27-1~deb10u1.
 [PASS] PCRE compiled with unicode support.
 [PASS] The temporary directory and its content are writable.
 [PASS] The public image directory and its content are writable.
 [PASS] The logs directory and its content are writable.
 [PASS] GD or Imagick extension is installed.
 [PASS] Intl extension is installed.
 [PASS] Mbstring extension is installed.

 Config files

 [PASS] The application config file is present
 [PASS] The passbolt config file is present

 Core config

 [FAIL] Debug mode is on.
  [HELP] Set debug = false; in config/passbolt.php
 [PASS] Cache is working.
 [PASS] Unique value set for security.salt
 [PASS] Full base url is set to https://pass.domain.eu
 [PASS] App.fullBaseUrl validation OK.
 [PASS] /healthcheck/status is reachable.

 SSL Certificate

 [PASS] SSL peer certificate validates
 [PASS] Hostname is matching in SSL certificate.
 [PASS] Not using a self-signed certificate

 Database

 [PASS] The application is able to connect to the database
 [PASS] 25 tables found
 [PASS] Some default content is present
 [PASS] The database schema up to date.

 GPG Configuration

 [PASS] PHP GPG Module is installed and loaded.
 [PASS] The environment variable GNUPGHOME is set to /var/lib/passbolt/.gnupg.
 [PASS] The directory /var/lib/passbolt/.gnupg containing the keyring is writable by the webserver user.
 [PASS] The server gpg key is not the default one
 [PASS] The public key file is defined in config/passbolt.php and readable.
 [PASS] The private key file is defined in config/passbolt.php and readable.
 [PASS] The server key fingerprint matches the one defined in config/passbolt.php.
 [PASS] The server public key defined in the config/passbolt.php (or environment variables) is in the keyring.
 [PASS] There is a valid email id defined for the server key.
 [PASS] The public key can be used to encrypt a message.
 [PASS] The private key can be used to sign a message.
 [PASS] The public and private keys can be used to encrypt and sign a message.
 [PASS] The private key can be used to decrypt a message.
 [PASS] The private key can be used to decrypt and verify a message.
 [PASS] The public key can be used to verify a signature.

 Application configuration

 [PASS] Using latest passbolt version (3.1.0).
 [PASS] Passbolt is configured to force SSL use.
 [PASS] App.fullBaseUrl is set to HTTPS.
 [PASS] Selenium API endpoints are disabled.
 [PASS] Search engine robots are told not to index content.
 [PASS] Registration is closed, only administrators can add users.
 [PASS] Serving the compiled version of the javascript app
 [PASS] All email notifications will be sent.

  1 error(s) found. Hang in there!


```

Thank you in advance for your help !

**Comment by stripthis** [comment]
Thanks for the report, we have identified the source of the problem and will be publishing a fix soon.

**Comment by stripthis** [comment]
Fixed in 3.3.
